webpack/assets/javascripts/react_app/redux/actions/common/forms.test.js

```diff
@@ -24,5 +24,8 @@ describe('form actions', () => {
     expect(() => {
       submitForm({ values: { a: 1 } });
     }).toThrow();
+    expect(() => {
+      submitForm({ customErrorAlert: 'Value should not be blank' });
+    }).toThrow();
```

Comment:
I think a better option is to pass the params to the action and get a snapshot of what it returns, e.g:

```
import { testActionSnapshotWithFixtures } from '@theforeman/test';

const fixtures = {
  'should submitForm': () => submitForm({
    item,
    url,
    values,
    message,
    customErrorAlert,
  }),
};

describe('Forms actions', () =>
  testActionSnapshotWithFixtures(fixtures));
```
In this PR it would be OK to add only your scenario,
though maybe some refactoring is needed here as a follow-up :)

Reply:
makes sense.